### ISSUE TITLE
Reports unused property while property is used in complex observers

### ISSUE THREAD

**Issue by TimvdLippe** [Polymer/polymer-linter#91]
I am getting the following error:

```
    <iron-media-query query="(max-width:432px)"  query-matches="{{mobile}}"></iron-media-query>
                                                                  ~~~~~~

lancie-dialog.html(111,66) warning [databind-with-unknown-property] - mobile is not declared or used more than once. Did you mean: title
```

With the following code (snippet):

```html

<template>
  <...>
  <iron-media-query query="(max-width:432px)"  query-matches="{{mobile}}"></iron-media-query>
  <...>
</template>
<script>
  'use strict';

  (function() {
    Polymer({
      is: 'lancie-dialog',

      observers: [
        '_viewChanged(mobile)',
...
</script>
```

You can see that the property is actually used in the observer, but still reported as unused.

**Comment by stramel** [comment]
@timvdlippe It is saying that `mobile` is not declared, not that it isn't used.

**Comment by TimvdLippe** [comment]
`or used more than once` For other properties where I have another binding it works. I do expect the same for properties I use in listeners/observers, etc...

**Comment by stramel** [comment]
@timvdlippe `used more than once` is still different than the error that you are describing in the original issue of being unused.

I'm not sure what you are saying is incorrect about the warning.

**Comment by stramel** [comment]
Ah after discussing with @timvdlippe and @usergenic, I understand what is going on. 

Tim would like the observer and I assume other usages of the non-declared properties within the script to count towards the usages which would allow this warning to go away.

I still feel like public and properties used in observers should be declared but maybe that's just me.
